packages/swapper/src/swappers/ChainflipSwapper/utils/getQuoteOrRate.ts

```diff
@@ -297,6 +297,13 @@ export const getQuoteOrRate = async (
         singleQuoteResponse.boostQuote.egressAmountNative!,
       )
 
+      const buyAmountBeforeFeesCryptoBaseUnit = toBaseUnit(
```

Comment:
suggestion: comment me to explain that this isn't a buy amount before fees per se, but rather an input/output calculation to get the sell amount prorated to the buy asset price, in an effort to determine price impact

Reply:
Changed in [0cec0f9](https://github.com/shapeshift/web/pull/8556/commits/0cec0f9aa5c1a9d6c47cad3ad5371369eff749ed)

As well as other line doing the same for boosted quote